chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -526,6 +529,7 @@ impl PeerManagerActor {
         &mut self,
         ctx: &mut Context<Self>,
         interval: time::Duration,
+        start_time: time::Instant,
```

Comment:
since you are using a clock in the function anyway, don't pass the timestamp argument, but rather call self.clock.now() within the function call.